api/src/main/java/io/github/sebastiantoepfer/jsonschema/keyword/StaticAnnotation.java

```diff
@@ -51,4 +54,21 @@ public boolean hasName(final String name) {
     public JsonValue valueFor(final JsonValue instance) {
         return value;
     }
+
+    @Override
+    public <T extends Media<T>> T printOn(final T media) {
+        final T result;
+        if (value instanceof JsonString str) {
+            result = media.withValue(name, str.getString());
+        } else if (value instanceof JsonNumber nr) {
+            result = media.withValue(name, nr.bigDecimalValue());
+        } else if (value == JsonValue.TRUE) {
+            result = media.withValue(name, true);
+        } else if (value == JsonValue.FALSE) {
+            result = media.withValue(name, false);
+        } else {
+            result = media;
+        }
+        return result;
```

Comment:
use JsonObjectPrintable:
```suggestion
        return new JsonObjectPrintable(Json.createObjectBuilder().add(name, value).build())
                .printOn(media);
```